notebooks/101-benchmarking/101-benchmarking.ipynb

```diff
@@ -0,0 +1,391 @@
+{
```

Comment:
<p>Maybe you could give a little bit explanation here how you've created this configuration file. Just a few sentences explaining that the parameters were copied from the default padim config in <code>anomalib/models/padim/config.yaml</code>, and which sections were changed.   Otherwise the reader might think that they need to set all these parameters manually.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/353/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>